fastlane/README.md

```diff
@@ -4,29 +4,16 @@ fastlane documentation
 
 Make sure you have the latest version of the Xcode command line tools installed:
 
-```no-highlight
+```
 xcode-select --install
 ```
 
-## Choose your installation method:
```

Comment:
Did we move this section somewhere else in a previous commit or something?

Reply:
[Yep](https://github.com/fastlane/docs/pull/525/files#diff-53f94788bbe9e3dae55249156d48441fL5), and I guess that no _fastlane_ lane was run in order for this to be generated again.

Reply:
Ha, 👍